components/src/test/java/org/folio/rest/workflow/model/DatabaseQueryTaskTest.java

```diff
@@ -226,4 +233,77 @@ void setIncludeHeaderWorksTest() {
     assertEquals(true, getField(databaseQueryTask, "includeHeader"));
   }
 
+  @ParameterizedTest
+  @MethodSource("providePrePersistFor")
+  void prePersistWorksTest(Map<String, Object> initial, Map<String, Object> expected) {
+    initial.forEach((String attribute, Object value) -> {
+      setField(databaseQueryTask, attribute, value);
+    });
+
+    databaseQueryTask.prePersist();
+
+    expected.forEach((String attribute, Object value) -> {
+      assertEquals(value, getField(databaseQueryTask, attribute));
+    });
+  }
+
+  /**
+   * Helper function for parameterized tests for the prePersist function.
+   *
+   * @return
+   *   The arguments array stream with the stream columns as:
+   *     - Arguments initial The initial values.
+   *     - Arguments expect The expected values.
+   */
+  private static Stream<Arguments> providePrePersistFor() {
+    final String designation = "designationValue";
+    final String query = "queryValue";
```

Comment:
Rather than define Strings here, instead use `VALUE`.
This avoids repreated code.
The only time we don't want to use `VALUE` is when specific meaning is needed or when distinction between two values are needed.

Reply:
String values now replaced by  VALUE (public static final String VALUE = "value";)